conans/server/conf/__init__.py

```diff
@@ -139,8 +139,8 @@ def write_permissions(self):
     def users(self):
         def validate_pass_encoding(password):
             try:
-                password.encode('latin1')
-            except UnicodeDecodeError:
+                password.encode('ascii')
```

Comment:
Sure that this change from latin1 to ascii will not have problems for users with passwords with latin1 only chars?